src/vueWrapper.ts

```diff
@@ -42,6 +45,21 @@ export class VueWrapper<T extends ComponentPublicInstance> {
     return this.vm.$el.parentElement
   }
 
+  private attachNativeEventListener(): void {
+    const vm = this.vm
+    if (!vm) return
+
+    const element = this.element
+    for (let key in element) {
+      if (/^on/.test(key)) {
+        const eventName = key.slice(2)
+        element.addEventListener(eventName, (...args) => {
+          recordEvent(vm.$, eventName, args)
+        })
+      }
```

Comment:
Pretty ugly way to add event listeners for all possible events, but it works.

Do these event listeners need to be removed on cleanup?

Reply:
Probably not, I am guessing when the element is cleaned up the listeners are as well

Reply:
We should use 

```ts

// @ts-ignore todo - No DefinitelyTyped package exists for this
import eventTypes from 'dom-event-types'

// ... 
    const element = this.element
    for (let eventName of Object.keys(eventTypes)) {
      element.addEventListener(eventName, (...args) => {
        recordEvent(vm.$, eventName, args)
      })
    }
```

This was is missing some events like compositionstart, and also seems to work differently in FF, chrome, jsdom etc.